rero_ils/modules/items/serializers/json.py

```diff
@@ -78,6 +88,15 @@ def _postprocess_search_hit(self, hit: dict) -> None:
         location = self.get_resource(LocationsSearch(), location_pid)
         metadata['location']['name'] = location.get('name')
 
+        # Try to serialize circulation information from best possible
+        # related `ItemType` resource if exists.
+        itty_key = 'temporary_item_type' if 'temporary_item_type' in metadata \
+            else 'item_type'
+        if itty_pid := metadata.get(itty_key, {}).get('pid'):
```

Comment:
You can also write like: `metadata.get('temporary_item_type', metadata.get('item_type'))['pid']`